internal/oci/container.go

```diff
@@ -168,6 +176,8 @@ func (c *Container) StatePath() string {
 
 // CreatedAt returns the container creation time
 func (c *Container) CreatedAt() time.Time {
+	c.stateLock.RLock()
```

Comment:
We don't need a read lock for this since this is set as part of `NewContainer` and then never modified.

Reply:
@mrunalp 
The read lock is taken out.